pandas_script.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # 1. Построить гистограммы
+
+# In[124]:
+
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+# In[54]:
+
+
+c = pd.read_csv('train.csv')
```

Comment:
Стоило положить этот файл в репозиторий или же открывать его прямо по ссылке
```suggestion
c = pd.read_csv('https://raw.githubusercontent.com/Serfentum/bf_course/master/14.pandas/train.csv')
```
`c` не самое хорошее имя для переменной